osi_datarecording.proto

```diff
@@ -27,16 +29,20 @@ message SensorDataList
 {
     // List of sensor data for multiple sensors at a specific timestep.
     //
-    repeated SensorData sensors = 1;
+    // \note OSI uses singular instead of plural for repeated field names.
```

Comment:
This is meant for the word "data" but not for everything else I guess?

Reply:
Maybe I misunderstood the discussion with Pierre?!
So we use definitive PLURAL? --> No problem! All field names (which have to change are marked with the same note).